sdk/documentintelligence/azure-ai-documentintelligence/azure/ai/documentintelligence/models/__init__.py

```diff
@@ -11,7 +11,7 @@
 from ._models import AnalyzeBatchOperationDetail
 from ._models import AnalyzeBatchResult
 from ._models import AnalyzeBatchResultOperation
-from ._models import AnalyzeDocumentRequest
+from ._patch import AnalyzeDocumentRequest
```

Comment:
Until we have a better way, I suggest adding the post-processing script to your pyproject.toml so we don't forget to generate with it. like this:

https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/translation/azure-ai-translation-document/pyproject.toml#L4-L5